packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/document/AddDocumentAssetFromXDDRequest.java

```diff
@@ -15,4 +15,6 @@ public class AddDocumentAssetFromXDDRequest {
 	private Document document;
 
 	private UUID projectId;
+
+	private String domain = "epi"; // default value
```

Comment:
This feels like something we [should] have an enum for

Reply:
Yes it should at the APP level. Let's table that for our talk on April 3